### PR TITLE
Dec issue v5

### PR DESCRIPTION
**Pull request by rishi9684**
#help component added

### CODE REVIEW

File: components/common/help/index.vue [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+            >
+              <i18n path="Common.ContactSupport" />
+            </v-btn>
+          </v-card-actions>
+        </div>
+        <div v-else>
+          <v-col class="col-12 col-md-12 py-0 mt-4">
+            <v-text-field
+              :value="this.$auth.$state.user.data.name"
+              label="Name"
```
Comment (SukantGujar) [inline components/common/help/index.vue:125]: Hardcoded

File: components/common/help/index.vue [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+              outlined
+              disabled
+              dense
+            ></v-text-field>
+          </v-col>
+          <v-col class="col-12 col-md-12 py-0">
+            <v-text-field
+              :value="this.$auth.$state.user.data.email"
+              label="Email"
```
Comment (SukantGujar) [inline components/common/help/index.vue:134]: Hardcoded, check further as well.

File: components/common/help/index.vue [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  </div>
+</template>
+
+<script>
+import nuxtconfig from '~/nuxt.config'
+export default {
+  props: {
+    helpDocs: {
+      type: String,
+      default: 'https://event.bitpod.io/admin/docs/HelpCenter/',
```
Comment (SukantGujar) [inline components/common/help/index.vue:179]: Move into localization

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: components/common/help/index.vue (hunk 1 above)
Comment (rishi9684) [reply components/common/help/index.vue]: resolve

**Review by SukantGujar — APPROVED** [review]

File: components/common/help/index.vue (hunk 3 above)
Comment (rishi9684) [reply components/common/help/index.vue]: move in locale

**Pull request merged**